docs/sdk-writer-guide.md

```diff
@@ -33,9 +33,9 @@ These SDK writer output contains just a carbondata and carbonindex files. No met
  
      Schema schema = new Schema(fields);
  
-     CarbonWriterBuilder builder = CarbonWriter.builder().withSchema(schema).outputPath(path);
```

Comment:
In the same document handle the below points. 
1. remove withSchema() interface 
2. update the below interfaces, 
buildWriterForCSVInput
buildWriterForAvroInput

Reply:
done